org.prismlauncher.PrismLauncher.yml

```diff
@@ -25,6 +18,7 @@ finish-args:
   - --filesystem=xdg-run/app/com.discordapp.Discord:create
     # Mod drag&drop
   - --filesystem=xdg-download:ro
+  - --env=PATH=$PATH:/usr/lib/extensions/vulkan/gamescope/bin
```

Comment:
I don't think adding `$PATH` here works.

Besides, from what I can see, we're already modifying `PATH` inside the `prismlauncher` script, so it probably makes sense to add this new path there as well.

Reply:
Oh, that we are. We can just change the gamescope path there. Will get to it later.